src/DB/product.json

```diff
@@ -1714,6 +1714,13 @@
   "image": "https://bit.ly/3P09nA5",
   "link": "https://novoresume.com/",
   "description": "Online resume builder."
+},
+{
+  "productName": "Uiverse.io",
+  "category": "UI",
+  "image": "https://tinyurl.com/uiverse-logo",
```

Comment:
image link should be shortened.
it is mention in the contribution.md 

```
To ensure that your URLs are short, it is recommended to use [bitly](https://bitly.com/) for URL shortening.
```

Reply:
The image URL has been changed